### ISSUE TITLE
[python] Latest version broke Python 3 compatibility

### ISSUE THREAD

**Issue by mati865** [jruizgit/rules#283]
This commit to be specific https://github.com/jruizgit/rules/commit/01509f228e04025fdff4b53d836e4ebcb2490c51

The error:
```
/project/path/venv/lib/python3.6/site-packages/rules.cpython-36m-x86_64-linux-gnu.so: undefined symbol: PyInt_AsLong
```

`PyInt_AsLong` is Python 2 function replaced by `PyLong_AsLong` in Python 3.

**Comment by jruizgit** [comment]
Thanks for reporting this issue. I have published a fix for it, please use version 2.0.13

**Comment by mati865** [comment]
Thank you for the quick fix.
